Rover/Rover.cpp

```diff
@@ -384,23 +381,31 @@ void Rover::update_mission(void)
     }
 }
 
-#if OSD_ENABLED == ENABLED
-void Rover::publish_osd_info()
+// vehicle specific waypoint info helpers
+bool Rover::get_wp_distance_m(float &distance) const
+{
+    distance = control_mode->get_distance_to_destination();
+    return true;
+}
+
+// vehicle specific waypoint info helpers
+bool Rover::get_wp_bearing_d(float &bearing) const
+{
+    bearing = control_mode->wp_bearing() * 100.0f;
```

Comment:
should be 0.01 ?

Reply:
control_mode->wp_bearing() it's already in degrees so no multiplier (OSD expects bearing in centidegrees though)